### PR TITLE
[Merged by Bors] - feat(LpSpace): `indicatorConstLp` is injective in set

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Function/LpSpace.lean [hunk 1]
```diff
@@ -822,11 +823,18 @@ theorem dist_indicatorConstLp_eq_norm {t : Set α} (ht : MeasurableSet t) (hμt
 @[simp]
 theorem indicatorConstLp_empty :
     indicatorConstLp p MeasurableSet.empty (by simp : μ ∅ ≠ ∞) c = 0 := by
-  rw [Lp.eq_zero_iff_ae_eq_zero]
-  convert indicatorConstLp_coeFn (E := E)
-  simp [Set.indicator_empty', Pi.zero_def]
+  simp only [indicatorConstLp, Set.indicator_empty', Memℒp.toLp_zero]
 #align measure_theory.indicator_const_empty MeasureTheory.indicatorConstLp_empty
 
+theorem indicatorConstLp_left_injective {s t : Set α} (hs : MeasurableSet s) (hsμ : μ s ≠ ∞)
```
[the remark this answers is not in the document]
Comment (urkud) [reply Mathlib/MeasureTheory/Function/LpSpace.lean:829]: There is also injectivity in `c`.

[the remark this answers is not in the document]
Comment (urkud) [reply Mathlib/MeasureTheory/Function/LpSpace.lean]: `s/_left//`

File: Mathlib/MeasureTheory/Function/LpSpace.lean [hunk 2]
```diff
@@ -835,11 +836,18 @@ theorem dist_indicatorConstLp_eq_norm {t : Set α} {ht : MeasurableSet t} {hμt
 @[simp]
 theorem indicatorConstLp_empty :
     indicatorConstLp p MeasurableSet.empty (by simp : μ ∅ ≠ ∞) c = 0 := by
-  rw [Lp.eq_zero_iff_ae_eq_zero]
-  convert indicatorConstLp_coeFn (E := E)
-  simp [Set.indicator_empty', Pi.zero_def]
+  simp only [indicatorConstLp, Set.indicator_empty', Memℒp.toLp_zero]
 #align measure_theory.indicator_const_empty MeasureTheory.indicatorConstLp_empty
 
+theorem indicatorConstLp_injective {s t : Set α} (hs : MeasurableSet s) (hsμ : μ s ≠ ∞)
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Function/LpSpace.lean:842]: Shouldn't this be spellt `indicatorConstLp_inj`? (I think the `injective` version is for when you use `Function.Injective ...`).

**Review by sgouezel — COMMENTED** [review]
bors d+

**Pull request closed without merging**